src/app/visual/readonly-grid/readonly-grid.component.html

```diff
@@ -1,11 +1,13 @@
 <div
   class="readonly-grid"
 >
-  <ag-grid-angular
-    [gridOptions]="gridOptions"
-    [rowData]="gridData"
-  >
-  </ag-grid-angular>
+  <sky-ag-grid-wrapper>
```

Comment:
Maybe not part of this PR, but I noticed there's no way to sort the columns with keyboard controls? Maybe I'm missing it.

Reply:
hmm I'll look into this, looks like ag-grid doesn't let you focus on the column headers 🤔